### PR TITLE
Merge qube TEMPLATE 

### PR DESCRIPTION
**Pull request by KochTobi**
This PR merges the TEMPLATE from `qube 2.6.1` into the development branch.

### CODE REVIEW

File: src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy [hunk 1]
```diff
@@ -178,7 +178,7 @@ class DependencyManager {
 
         ControlElements sampleModifyControls
         try {
-            def userEmail = PortalUtils.isLiferayPortlet() ? PortalUtils.getUser().getEmailAddress() : "Not logged in."
+            def userEmail = PortalUtils.isLiferayPortlet() ? PortalUtils.getUser().getEmailAddress() : "tobias.koch@qbic.uni-tuebingen.de"
```
Comment (Steffengreiner) [inline src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy:181]: I'd remove your email if that is alright 
```suggestion
            def userEmail = PortalUtils.isLiferayPortlet() ? PortalUtils.getUser().getEmailAddress() : "Not logged in"
```

File: .github/workflows/generate-reports.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install git
+        run: sudo apt-get install git
+
+      - name: Set up JDK 1.11
```
Comment (Steffengreiner) [inline .github/workflows/generate-reports.yml:23]: I think we should keep the JDK version consistent for all github actions 
```suggestion
      - name: Set up JDK 1.8
```

File: .github/workflows/generate-reports.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (Steffengreiner) [inline .github/workflows/generate-reports.yml:26]: I think we should keep the JDK version consistent for all github actions 
```suggestion
          java-version: 1.8
```

File: .github/workflows/nexus-publish-release.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
```
Comment (Steffengreiner) [inline .github/workflows/nexus-publish-release.yml:18]: I think we should keep the JDK version consistent for all github actions 
```suggestion
    - name: Set up JDK 1.8
```

File: .github/workflows/nexus-publish-release.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
```
Comment (Steffengreiner) [inline .github/workflows/nexus-publish-release.yml:21]: TL:DR: The suggestion keeps moving on 
```suggestion
        java-version: 1.8
```

File: .github/workflows/nexus-publish.yml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      - development
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
```
Comment (Steffengreiner) [inline .github/workflows/nexus-publish.yml:19]: Something Something Java 1.8
```suggestion
      - name: Set up JDK 1.8
```

File: .github/workflows/nexus-publish.yml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (Steffengreiner) [inline .github/workflows/nexus-publish.yml:22]: I think we should keep the JDK version consistent for all github actions 
```suggestion
          java-version: 1.8
```

File: .github/workflows/run_tests.yml [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
```
Comment (Steffengreiner) [inline .github/workflows/run_tests.yml:11]: I think we should keep the JDK version consistent for all github actions 
```suggestion
      - name: Set up JDK 1.8
```

File: .github/workflows/run_tests.yml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (Steffengreiner) [inline .github/workflows/run_tests.yml:14]:
```suggestion
          java-version: 1.8
```

File: CHANGELOG.rst [hunk 10]
```diff
@@ -0,0 +1,19 @@
+==========
+Changelog
+==========
+
+This project adheres to `Semantic Versioning <https://semver.org/>`_.
+
+
+1.2.0-SNAPSHOT (2021-02-08)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:8]: Should the Changelog be updated here or do you want to do it in a seperate Branch?

File: pom.xml [hunk 11]
```diff
@@ -1,21 +1,35 @@
...
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>portlet-parent-pom</artifactId>
-		<version>2.1.0</version>
+		<version>3.1.3</version>
 	</parent>
```
Comment (Steffengreiner) [inline pom.xml:11]: AFAIK we wanted to remove the parent pom from our products. I think this should be done in a seperate PR however

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Looks Good, However some stuff is unclear to me and i would recommend sticking to one JDK Version for all the Github actions.

File: pom.xml (hunk 11 above)
Comment (KochTobi) [reply pom.xml:11]: For now I only used the most recent version of the qube template

File: CHANGELOG.rst (hunk 10 above)
Comment (KochTobi) [reply CHANGELOG.rst:8]: I would suggest doing that in a different branch. This PR should focus on integrating the TEMPLATE branch into the project.

**Review by Steffengreiner — APPROVED** [review]
LGTM!

**Pull request merged**